common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -1154,3 +1154,29 @@ def test_user_not_enrolled_error(self):
             u"{user} is not enrolled in this course. Please check your spelling and retry.".format(user=new_user),
             self.certificates_section.certificate_invalidation_message.text
         )
+
+@attr('a11y')
+class LMSInstructorDashboardA11yTest(BaseInstructorDashboardTest):
+    """
+    LMS Instructor Dashboard Accessibility Test Class
+    """
+
+    def setUp(self):
+        browser = os.environ.get('SELENIUM_BROWSER', 'firefox')
+
+        with patch.dict(os.environ, {'SELENIUM_BROWSER': browser}):
+            super(LMSInstructorDashboardA11yTest, self).setUp()
+
+        self.course_fixture = CourseFixture(**self.course_info).install()
+        self.log_in_as_instructor()
+        self.instructor_dashboard_page = self.visit_instructor_dashboard()
+
+    def test_instructor_dashboard_a11y(self):
+        # a11y test added on 17 May 2016, but mostly disabled as there's a slew 
+        # of issues on the Instructor Dashboard. We should aim to resolve these 
+        # before 2 October 2016, and include the below exclusions.
+        self.instructor_dashboard_page.a11y_audit.config.set_scope(
+            include=[],
+            exclude=["table", "a"]
```

Comment:
These failures aren't included in the scope of this work, so I'm excluding them from running. However, we should tackle these soon to ensure proper accessibility on this page.